rpc/types/src/bin.rs

```diff
@@ -91,15 +94,29 @@ define_request_and_response! {
     }
 }
 
+#[cfg(feature = "epee")]
+define_request_and_response! {
+    get_o_indexesbin,
+    cc73fe71162d564ffda8e549b79a350bca53c454 =>
+    core_rpc_server_commands_defs.h => 487..=510,
+    GetOutputIndexes,
+    #[derive(Copy)]
+    Request {
+        txid: [u8; 32],
+    },
+    AccessResponseBase {
+        o_indexes: Vec<u64> as ContainerAsBlob<u64>,
+    }
+}
```

Comment:
Not ideal: there are 2 definitions for these `as` types. If `epee`:
```rust
o_indexes: Vec<u64> as ContainerAsBlob<u64>,
```
else:
```rust
o_indexes: Vec<u64>,
```

Reply:
yeah that's not great, if this is the only type then it's probably acceptable